falcon/status_codes.py

```diff
@@ -90,6 +90,12 @@
 HTTP_IM_A_TEAPOT = HTTP_418
 HTTP_426 = '426 Upgrade Required'
 HTTP_UPGRADE_REQUIRED = HTTP_426
+HTTP_428 = '428 Precondition Required'
```

Comment:
It would be nice to add some subclasses of HTTPError that correspond to these...